src/test/javascript/spec/component/course/course-group.component.spec.ts

```diff
@@ -75,130 +68,145 @@ describe('Course Management Detail Component', () => {
             });
-            expect(comp.searchFailed).to.be;
-            expect(searchStub).to.have.been.calledWith(loginOrName);
+            expect(comp.searchFailed).toBe(true);
+            expect(searchStub).toHaveBeenCalledWith(loginOrName);
         });
     });
 
     describe('onAutocompleteSelect', () => {
-        let addUserStub: sinon.SinonStub;
-        let fake: sinon.SinonSpy;
+        let addUserStub: jest.SpyInstance;
+        let fake: jest.Mock;
         let user: User;
 
         beforeEach(() => {
-            addUserStub = sinon.stub(courseService, 'addUserToCourseGroup');
-            addUserStub.returns(of(new HttpResponse()));
-            fake = sinon.fake();
+            addUserStub = jest.spyOn(courseService, 'addUserToCourseGroup').mockImplementation();
+            addUserStub.mockReturnValue(of(new HttpResponse()));
+            fake = jest.fn();
             user = courseGroupUser;
             comp.allCourseGroupUsers = [];
             comp.course = course;
             comp.courseGroup = courseGroup;
         });
+
         it('should add the selected user to course group', () => {
             comp.onAutocompleteSelect(user, fake);
-            expect(addUserStub).to.have.been.calledWith(course.id, courseGroup, user.login);
-            expect(comp.allCourseGroupUsers).to.deep.equal([courseGroupUser]);
-            expect(fake).to.have.been.calledWithExactly(user);
+            expect(addUserStub).toHaveBeenCalledWith(course.id, courseGroup, user.login);
+            expect(comp.allCourseGroupUsers).toEqual([courseGroupUser]);
+            expect(fake).toHaveBeenCalledWith(user);
         });
+
         it('should call callback if user is already in the group', () => {
             comp.allCourseGroupUsers = [user];
             comp.onAutocompleteSelect(user, fake);
-            expect(addUserStub).not.to.have.been.called;
-            expect(comp.allCourseGroupUsers).to.deep.equal([courseGroupUser]);
-            expect(fake).to.have.been.calledWithExactly(user);
+            expect(addUserStub).not.toHaveBeenCalled();
+            expect(comp.allCourseGroupUsers).toEqual([courseGroupUser]);
+            expect(fake).toHaveBeenCalledWith(user);
         });
     });
 
     describe('removeFromGroup', () => {
-        let removeUserStub: sinon.SinonStub;
+        let removeUserStub: jest.SpyInstance;
 
         beforeEach(() => {
-            removeUserStub = sinon.stub(courseService, 'removeUserFromCourseGroup');
-            removeUserStub.returns(of(new HttpResponse()));
+            removeUserStub = jest.spyOn(courseService, 'removeUserFromCourseGroup').mockImplementation();
+            removeUserStub.mockReturnValue(of(new HttpResponse()));
             comp.allCourseGroupUsers = [courseGroupUser, courseGroupUser2];
             comp.course = course;
             comp.courseGroup = courseGroup;
         });
+
+        afterEach(() => {
+            jest.restoreAllMocks();
```

Comment:
Can be removed

Reply:
I removed it. Thanks for pointing this out.